android/assets/jsons/Civ V - Vanilla/Buildings.json

```diff
@@ -390,7 +390,7 @@
 		"culture": 1,
 		"isWonder": true,
 		"greatPersonPoints": {"Great Engineer": 1},
-		"uniques": ["Must be next to [Desert]", "[+1 Food, +1 Production] from [Desert] tiles [in this city]", "[-1 Food, -1 Production] from [Flood plains] tiles [in this city]", "[+6 Culture] once [Archaeology] is discovered"],
+		"uniques": ["Must be next to [Desert]", "[+1 Food, +1 Production, +1 Gold] from [Desert] tiles [in this city]", "[-1 Food, -1 Production, -1 Gold] from [Flood plains] tiles [in this city]", "Provides a free [Amphitheater] [in this city]"], "[+6 Culture] once [Archaeology] is discovered"],
```

Comment:
There's an extra ] at the end of "in this city" that's causing the test errors

Reply:
ok